cmd/object-api-utils.go

```diff
@@ -597,3 +598,39 @@ func (g *GetObjectReader) Read(p []byte) (n int, err error) {
 	}
 	return
 }
+
+// PutObjectReader is a type that wraps sio.EncryptReader and
+// underlying hash.Reader in a struct
+type PutObjectReader struct {
```

Comment:
This type should not really end with the Reader suffix as it is not an `io.Reader`. Maybe `PutObjReaders` or something else.